widget/embedded/src/lib/tabManager/tabManager.ts

```diff
@@ -0,0 +1,235 @@
+import type {
+  Events,
+  ID,
+  Payload,
+  TabManagerInterface,
+} from './tabManager.types';
+
+import { CHANNEL_NAME, TIMEOUT } from './tabManager.constants';
+
+export class TabManager implements TabManagerInterface {
+  private id: ID;
```

Comment:
Is this the active tab id? if so,  we could change  it to activeTabId or something like that.